index.html

```diff
@@ -0,0 +1,282 @@
+          <p>
+            A smart assistant to make driving more safe, efficient, and fun by
+            unlocking your most expensive computer: your car.
+          </p>
+          <menu>
+            <li>HTML</li>
+            <li>CSS</li>
+            <li>Ruby on rails</li>
+            <li>JavaScript</li>
+          </menu>
+          <a class="btn-action">See Project</a>
+        </div>
+      </div>
+
+      <div class="card">
+        <div class="card-img">
+          <img src="./images/Snapshoot-Portfolio-2.png" alt="Tonic" />
+        </div>
+
+        <div class="details">
+          <h2>Multi-Post Stories</h2>
+          <ul>
+            <li>CANOPY</li>
+            <li>Back End Dev</li>
+            <li>2015</li>
+          </ul>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <menu>
+            <li>HTML</li>
+            <li>CSS</li>
+            <li>Ruby on rails</li>
+            <li>JavaScript</li>
+          </menu>
+          <a class="btn-action">See Project</a>
+        </div>
+      </div>
+
+      <div class="card">
+        <div class="card-img">
+          <img src="./images/Snapshoot-Portfolio-3.png" alt="Tonic" />
+        </div>
+
+        <div class="details">
+          <h2>Multi-Post Stroies</h2>
+          <ul>
+            <li>FACEBOOK</li>
+            <li>FULL Stack Dev</li>
+            <li>2015</li>
+          </ul>
+          <p>
+            Experimental content creation feature that allows users to add to
+            an existing story over the course of a day without spamming their
+            friends.
+          </p>
+          <menu>
+            <li>html</li>
+            <li>css</li>
+            <li>Ruby on rails</li>
+            <li>javaScript</li>
+          </menu>
+          <a class="btn-action">See Project</a>
+        </div>
+      </div>
+    </section>
+  </div>
+    <section id="about-me">
+      <div id="about-myself">
+        <h3>About<br> mySelf</h3>
+        <p>
+          I'm a software developer! I can help you build a product , feature
+          or website Look through some of my work and experience! If you like
+          what you see and have a project you need coded, don't hestiate to
+          contact me.
+        </p>
+
+        <a class="pointer btn-action">LET'S CONNECT</a>
+
+        <ul class="socials">
+          <li class="pointer">
+            <img src="./images/twitter.svg" alt="Twitter" />
+          </li>
+          <li class="pointer">
+            <img src="./images/Linkedin.svg" alt="Linkedin" />
+          </li>
+          <li class="pointer">
+            <img src="./images/medium.svg" alt="Medium" />
+          </li>
+          <li class="pointer">
+            <img src="./images/github.svg" alt="Github" />
+          </li>
+          <li class="pointer">
+            <img src="./images/angellist.svg" alt="Angellist" />
+          </li>
+        </ul>
```

Comment:
- You did a great job in the organization of your code it looks really clean however there is just a small thing you still need to fix. If the section has a navigation link (e.g., "Work") it uses an `a ` tag. kindly for the  list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag)  as the icons must have the navigation links :+1: